### DISCUSSION TITLE
USR W610

### DISCUSSION THREAD

**Discussion by djdarlo** [nathanmarlor/foxess_modbus#442] [Support] [answered]
I've been using this integration for months without any issues until today.  I'm using a USR W610 (UDP) which is connected to the network fine.  I can see it connected to my wifi on 192.168.1.102.  It seems to stop communicating with HA after a number of minutes.  If I restart the USR W610 it works again for a few minutes then I get the message saying entity is currently unavailable.  When I try to check the configuration in the integration menu I get a message saying...

Error connecting to your adapter or inverter. Ensure the adapter is properly configured and is correctly wired to your inverter (see the setup link above), then try again. Details: Modbus Error: [Input/Output] timed out; Connection to (192.168.1.102, 502) failed: timed out; Connection to (192.168.1.102, 502) failed: [Errno 113] Host is unreachable

Any ideas?

**Comment by canton7** [comment]
"Host is unreachable" is a routing problem I believe, rather than something wrong with your W610. Is your HA on the same subnet as the W610?

**Comment by djdarlo** [comment, marked as answer]
It's a weird one isn't it.  I had this issue before and it was something to
do with my connection from HA - thanks for jogging my memory :) Might be
related to the BT wifi disc I'm running the RPI off.  Let me try a few
things and get back to you.

D
